CHANGELOG.md

```diff
@@ -13,6 +13,8 @@
   - Fix honoring of `includeCurrentStateVariables` flag in `WorkflowInstanceService.listWorkflowInstances`. This caused major slowness when using bulk workflows.
     To preserve the existing (incorrect) default behaviour in backwards compatible way the default value in `QueryWorkflowInstances.Builder` is changed to `true`. The REST API is unaffected.
     Especially in workflows with many children that use the `StateExecution.getAllChildWorkflows` method the performance impact can be high. Before 7.0.0 release, it is recommended to use `StateExecution.queryChildWorkflows(new QueryWorkflowInstances.Builder().setIncludeCurrentStateVariables(false).build())` if state variables are not needed.
+  - Improve child workflow final state execution speed by caching the parent workflow type. Use `nflow.db.type.cache.size` property to tune the size.
```

Comment:
Rename conf property to nflow.workflow.instance.type.cache.size or nflow.workflowInstanceType.cacheSize or something? db type cache size could be anything :)